tdrs-backend/tdpservice/settings/common.py

```diff
@@ -326,7 +326,7 @@ class Common(Configuration):
     logger.debug("RAW_CLAMAV: " + str(RAW_CLAMAV))
     CLAMAV_NEEDED = bool(strtobool(RAW_CLAMAV))
 
-    # The URL endpoint to send AV scan requests to (clamav-rest)
+    # The URL endpoint to send AV scan requests to (clamav-rest/clamav-nginx-proxy)
     AV_SCAN_URL = os.getenv('AV_SCAN_URL')
```

Comment:
I am not clear how you intend to add AV_SCAN_URL env var for each instance. Note that this is a dynamic env var and can not be imported in deploy backend script